examples/standalone/benchmarks/plotting.py

```diff
@@ -27,12 +27,12 @@
                 with open(fullpath) as f:
                     alldata.append(json.load(f))
     alldata.sort(
-        key=lambda k: datetime.strptime(
-            k["setup"]["experiment time"], "%d/%m/%Y %H:%M:%S"
-        )
+        key=lambda k: datetime.strptime(k["setup"]["timestamp"], "%d/%m/%Y %H:%M:%S")
     )
-    for plottype in ["mainLoop", "initTotal"]:
-        keyval = ["main_loop"] if plottype == "mainLoop" else ["init", "total"]
+    for plottype in ["mainLoop", "initializationTotal"]:
+        keyval = (
+            ["main_loop"] if plottype == "mainLoop" else ["initialization", "total"]
+        )
```

Comment:
Confused about what this does, can you add an intention-describing comment or use a variable name for `keyval` that reflects it?

e.g. something to do with subplots or line_names or whatever it actually is?

Reply:
Gonna deputize @twicki on this one